controller/src/main/java/org/jboss/as/controller/SimpleOperationDefinitionBuilder.java

```diff
@@ -201,9 +202,13 @@ public SimpleOperationDefinitionBuilder addAccessConstraint(final AccessConstrai
         return this;
     }
 
-    SimpleOperationDefinitionBuilder setDescriptionProvider(DescriptionProvider provider){
+    SimpleOperationDefinitionBuilder setDescriptionProvider(DescriptionProvider provider) {
         this.descriptionProvider = provider;
         return this;
     }
 
+    SimpleOperationDefinitionBuilder setStream(Stream stream) {
+        this.stream = stream;
```

Comment:
Should this be something like:
```java
this.stream = stream == null ? Stream.DEFAULT : stream;
```

Reply:
Oops - that code is obsolete.  Originally, I intended to associate Resource/Attribute/OperationDefinition instances with a stream, and conditionally register based on the stream used by the server - but that proved to be super awkward for ResourceDefinition registration, which expects a child MRR in return.